### ISSUE TITLE
Can we have some comparative voice samples ?

### ISSUE THREAD

**Issue by anjishnu** [tomlepaine/fast-wavenet#4]
Just to compare against Google's implementation.

**Comment by tomlepaine** [comment]
This repo doesn't focus on training on a large dataset of voice samples. We tried to convey this in the README and demo notebook, but we may have failed. It is just about a fast generation algorithm for wavenet. In the demo you can see the model train on a single audio sample, and generate a sample. It takes about 7 minutes on a Titan GPU (for both training and generation).

We're working on porting it to `tensorflow-wavenet`. Still verifying that our results match the original implementation. Should be ported soon.

**Comment by armator** [comment]
hello @tomlepaine. a little bit off-topic, but, do you own the gpu or use it from a cloud provider? do you have a suggestion what to buy etc? (It took about half a day for training and generation on my macbook)

**Comment by CodeHatchling** [comment]
@tomlepaine I'm afraid I apparently don't fully understand what you meant in the README either, and why this negates a comparison.

My understanding so far:

**Fast-wavenet** improves on **wavenet** by caching and reusing previous calculations. Fast-wavenet skips steps performed by wavenet, but because these steps are **"redundant"** the results **should be identical**, if not very similar. However, **contradictorily**, for some unspecified reason fast-wavenet differs from wavenet in a way that makes it unsuitable for traning on large datasets, and thus their results are not comparable.

Basically, I gathered from the readme that it's basically just a faster version of wavenet. So yes, why a comparison to wavenet is not appropriate is a bit of a puzzler.

**Comment by tomlepaine** [comment]
@CodeHatchling It differs in that this repo is focused on fast generation. It is possible to test that generation is correct, and fast, without training on a large dataset (see the demo).

Our goal is not recreating the results of the wavenet paper: i.e. training the model on a sufficiently large dataset for a sufficiently long time to produce a model that is capable of generating good novel samples.

`tensorflow-wavenet`, which we are collaborating with, is currently working on this. See this [issue](https://github.com/ibab/tensorflow-wavenet/issues/47).
